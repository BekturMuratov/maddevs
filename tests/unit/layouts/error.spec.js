import { shallowMount } from '@vue/test-utils'
import Error from '@/layouts/error.vue'

const stubs = ['NuxtLink']

const mocks = {
  $getMediaFromS3: image => image,
}

describe('Error layout', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(Error, {
      stubs,
      mocks,
    })
  })

  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
