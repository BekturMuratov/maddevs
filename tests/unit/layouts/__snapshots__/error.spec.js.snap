// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error layout renders correctly 1`] = `
<div
  class="error-page container"
>
  <lottiemad-stub
    autoplay="true"
    class="case_lottie"
    height="170px"
    id="404-code"
    lottielink="/images/Cases/error/lottie/404.json"
    width="100%"
  />
   
  <h1
    class="error-page_title"
  >
    
    The page you were looking for doesnâ€™t exist.
  
  </h1>
   
  <nuxtlink-stub
    class="error-page_link"
    to="/"
  >
    
    Back to our site
  
  </nuxtlink-stub>
</div>
`;
